src/api/index.ts

```diff
@@ -0,0 +1,10 @@
+import { downloadData } from 'aws-amplify/storage';
+import type { RunDetails } from '../sample-data/interfaces';
+
+// eslint-disable-next-line import/no-unused-modules
+export async function fetchMulitJSON():  Promise<Record<string, RunDetails[]>>{
+  const downloadResult = await downloadData({ key: "multi.json"  }).result;
+  const text = await downloadResult.body.text();
+  const json=JSON.parse(text) as Record<string, RunDetails[]>
```

Comment:
can not directly use  json response  as mentioned 

https://docs.amplify.aws/javascript/build-a-backend/storage/download/#get-the-text-value-of-downloaded-file

<img width="479" alt="image" src="https://github.com/surcyf123/pretraining-frontend/assets/25290212/348e5125-282b-4875-90ff-1143566de2bb">